wsmatrix@martin.zurowietz.de/overview/controlsManagerLayout.js

```diff
@@ -3,7 +3,8 @@ const Self = ExtensionUtils.getCurrentExtension();
 const Util = Self.imports.util;
 const OverviewControls = imports.ui.overviewControls;
 
-const { SMALL_WORKSPACE_RATIO } = OverviewControls;
+//const { SMALL_WORKSPACE_RATIO } = OverviewControls;
+const SMALL_WORKSPACE_RATIO = 0.15;
```

Comment:
Wait for Gnome Team to agree or disagree on opening this and other bits.

Reply:
> Wait for Gnome Team to agree or disagree on opening this and other bits.

Did you open an issue or merge request at https://gitlab.gnome.org/GNOME/gnome-shell?

Reply:
[Here](https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2266)

Or just [a patch](https://github.com/mzur/gnome-shell-wsmatrix/files/8458984/Open_JS_UI_bits.txt) (apply with patch -p1).